libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -185,6 +190,7 @@ public void storeMobilePolicyForOrg(UserAccount account, int timeout, int passLe
         final Editor e = sp.edit();
         e.putInt(KEY_TIMEOUT, timeout);
         e.putInt(KEY_PASSCODE_LENGTH, passLen);
+        e.putBoolean(KEY_BIOMETRIC_ALLOWED, bioAllowed);
```

Comment:
You'll need an upgrade step to add the default value here with the value `true`. Wait for my PR to go in before adding that though - you'll get the upgrade module for free.